app/javascript/scavenger_hunt/scenes/Questions/QuestionThree/QuestionThree.jsx

```diff
@@ -5,16 +5,34 @@ import DatePicker from "react-datepicker";
 import './QuestionThree.scss';
 import "react-datepicker/dist/react-datepicker.css";
 
+const formatedDate = (date) => {
+  return date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear(); 
+}
+
 const  QuestionThree = () => {
   const [DateOne, setDateOne] = useState(new Date());
   const [DateTwo, setDateTwo] = useState(new Date());
-
+  const [DateOneString, setDateOneString] = useState(formatedDate(new Date()));
+  const [DateTwoString, setDateTwoString] = useState(formatedDate(new Date()));
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    console.log("Answer Submitted is: ", DateOne, DateTwo);
+    console.log("Answer Submitted is: ", DateOneString, DateTwoString);
   }
 
+  const handleChange = (date, isFirstDate = false, isSecondDate = false) => {
+    let formated_date = formatedDate(date)
+    console.log(formated_date);
+    if (isFirstDate) { 
+      setDateOne(date);
+      setDateOneString(formated_date); 
+    }
+    if (isSecondDate) {
+      setDateTwo(date);
+      setDateTwoString(formated_date);
```

Comment:
Do we actually need to define two more variables to store the new format string value? We could apply the ```formatedDate``` directly in the ```handleSubmit```

Reply:
Ok, so I am  removing these two extra variables, and when @prairir works on the onSubmit function he can simply use the `formatedDate` function in there to get the formatted version (dd/mm/yyyy) of the  Date